extensions/amp-story/1.0/amp-story-access.css

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-amp-story-access {
+amp-story-access.i-amphtml-element {
   position: absolute !important;
-  display: flex !important;
+  display: flex; /** No !important to allow the runtime to hide/show it. */
```

Comment:
This will also let publishers show/hide it, even if accidentally ☹️

Is there another class or attribute that gets applied when hidden that we can write a rule using `!important` for?

Reply:
Done